tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -0,0 +1,207 @@
+# contrasting of conditions (then the depth bias will cancel out).
+# This parameter can also be set to ``'nai'`` (Neural Activity Index, see [1]_)
+# which implements
+# a further normalization with the estimated noise. An alternative way to take
+# care of this depth bias is to use the ``depth`` parameter and normalize the
+# forward solution instead of the weights. Note that if you compare conditions,
+# the depth bias will cancel out and it is possible to set both parameters to
+# ``None``.
+#
+# Furthermore, we will optimize the orientation of the sources such that output
+# power is maximized. This is achieved by setting ``pick_ori='max-power'``.
+# This gives us one source estimate per source (i.e., voxel), which is known
+# as a scalar beamformer. It is also possible to compute a vector beamformer,
+# which gives back three estimates per voxel, corresponding to the three
+# directions of the source. This can be achieved by setting
+# ``pick_ori='vector'`` and will yield a :class:`volume vector source estimate
+# <mne.VolVectorSourceEstimate>`.
+
+filters = make_lcmv(evoked.info, forward, data_cov, reg=0.05,
+                    noise_cov=noise_cov, pick_ori='max-power',
+                    weight_norm='unit-noise-gain', rank=None)
+
+# You can save the filter for later use with:
+# filters.save('filters-lcmv.h5')
+
+###############################################################################
+# Apply the spatial filter
+# ------------------------
+# The spatial filter can be applied to different data types: raw, epochs,
+# evoked data or the data covariance matrix to gain a static image of power.
+# The function to apply the spatial filter to :class:`~mne.Evoked` data is
+# :func:`~mne.beamformer.apply_lcmv` which is
+# what we will use here. The other functions are
+# :func:`~mne.beamformer.apply_lcmv_raw`,
+# :func:`~mne.beamformer.apply_lcmv_epochs`, and
+# :func:`~mne.beamformer.apply_lcmv_cov`.
+
+stc = apply_lcmv(evoked, filters, max_ori_out='signed')
+
+###############################################################################
+# Visualize the reconstructed source activity
+# -------------------------------------------
+# We can visualize the source estimate in different ways, e.g. as an overlay
+# onto the MRI or as a glass brain.
+# The plots show brain activity in the right temporal lobe around 100 ms post
+# stimulus. This is coherent with the left-ear auditory stimulation of the
+# experiment.
+
+lims = [0.3, 0.45, 0.6]
+
+stc.plot(
+    src=forward['src'], subject='sample', subjects_dir=subjects_dir,
+    clim=dict(kind='value', pos_lims=lims), mode='stat_map',
+    initial_time=0.087, verbose=True)
+###############################################################################
+stc.plot(
+    src=forward['src'], subject='sample', subjects_dir=subjects_dir,
+    mode='glass_brain', clim=dict(kind='value', lims=lims),
+    initial_time=0.087, verbose=True)
```

Comment:
Since the arguments are mostly overlapping, it might be helpful to extract what is common into a dict, and use kwarg unpacking. That would emphasize how easy it is to change from one visualization to the other. Example:

```
kwargs = dict(src=forward['src'], subject='sample', subjects_dir=subjects_dir,
              initial_time=0.087, verbose=True)

stc.plot(mode='stat_map', clim=dict(kind='value', pos_lims=lims), **kwargs)

###############################################################################

stc.plot(mode='glass_brain', clim=dict(kind='value', lims=lims), **kwargs)
```